examples/rbd/storageclass.yaml

```diff
@@ -29,6 +29,9 @@ parameters:
    # eg: pool: rbdpool
    pool: <rbd-pool-name>
 
+   # Set thickProvision to true if you want RBD images to be fully allocated on
+   # creation (thin provisioning is the default).
+   thickProvision: "false"
```

Comment:
Need to document this at https://github.com/ceph/ceph-csi/blob/master/docs/deploy-rbd.md

Reply:
ok, will add that!